spec/test-data/translations/glotpress-download/expected/values-pt-rBR/strings.xml

```diff
@@ -10,13 +10,12 @@ Language: pt_BR
     <string name="shipping_label_payments_saving_dialog_message">Aguarde…</string>
     <string name="shipping_label_payments_saving_dialog_title">Salvando suas configurações</string>
     <string name="shipping_label_payments_expiration_date">Expira em %1$s</string>
-    <string name="shipping_label_payments_email_receipts_checkbox">Enviar os recibos de compra da etiqueta por e-mail para %1$s (%2$s) em %3$s</string>
+    <string name="shipping_label_payments_email_receipts_checkbox">Enviar os recibos de compra da etiqueta por e&amp;#8211;mail para %1$s (%2$s) em %3$s</string>
```

Comment:
this is also a compound word, not a range, so the use of the hyphen was correct

Reply:
I misunderstood the linter error here a bit and was too quick to just overwrite here, I'll undo this 👍